src/work-allocation/containers/available-tasks/available-tasks.component.ts

```diff
@@ -15,6 +17,9 @@ export class AvailableTasksComponent extends TaskListWrapperComponent {
   private selectedLocations: Location[];
 
   public get fields(): TaskFieldConfig[] {
+    if (this.currentFeature === 'WorkAllocationRelease2') {
```

Comment:
'WorkAllocationRelease2'? Really? 😬

Reply:
it's what comes back from the mock service. I know i know.